managed-databases/postgresql-and-mysql/how-to/enable-autobackup.mdx

```diff
@@ -3,44 +3,50 @@ meta:
   title: How to enable autobackup for Databases
   description: This page explains how to enable autobackup for Databases
 content:
-  h1: How to enable autobackup for Databases
+  h1: How to manage autobackups for Database Instances
   paragraph: This page explains how to enable autobackup for Databases
-tags: managed-dabatase autobackup database 
+tags: managed-dabatase autobackup database
 dates:
-  validation: 2023-01-19
+  validation: 2023-04-18
   posted: 2019-05-11
 categories:
   - managed-databases
   - postgresql-and-mysql
 ---
 
-Backups are logical data dumps of an individual database. They are useful when you need to restore a specific logical database. 
+With the autobackup feature the creation and management of your Database Instance backups is automated according to your backup schedule configuration.
 
-For Database Instances that use Block Storage volumes, Managed Database for PostgreSQL and MySQL uses the [Snaphots](/managed-databases/postgresql-and-mysql/how-to/manage-snapshots/) feature to create autobackups. When the autobackup feature is enabled, Managed Database for PostgreSQL and MySQL create and manage backups according to your backup schedule configuration. 
+Two types of autobackups are available at Scaleway: [backups](/managed-databases/postgresql-and-mysql/concepts/#database-backup) and [snapshots](/managed-databases/postgresql-and-mysql/concepts/#database-snapshot).
+
+Backups are used when your Database Instance's storage option is a local volume. Snapshots are available for Database Instances that use [Block Storage volumes.
 
 <Macro id="iam-requirements" />
- 
+
 <Message type="requirement">
   - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
   - You have a [PostgreSQL or MySQL Database Instance](/managed-databases/postgresql-and-mysql/quickstart)
 </Message>
 
-
   <Message type="note">
-    The autobackup feature is enabled by default upon the creation of an Instance. Simultaneously, a backup schedule policy is created. The default backup frequency is of 1 per day and the retention period of 7 days. You can change this default configuration after you create the Instance from the database information page.
+    The autobackup feature is enabled by default upon the creation of an Instance. Simultaneously, a schedule policy is created. The default autobackup frequency is of 1 per day and the retention period of 7 days. You can change this in the Database Instance information page after Instance creation.
```

Comment:
Can we add something about the sheduling. 
By default we'll use the instance creation date as base timestamp for automated backup. 

When a user modify its configuration, it will be based on the current timestamp as well. 

i.e updating the backup schedule at 8am with frenquency set to 24h will run the backup the next morning at 8am